### PR TITLE
Add admin town command upkeep toggle

### PR DESCRIPTION
**Pull request by Gamer153**
#### Description: 

Towns can overwrite the upkeep setting of the config, but this setting wasn't modifyable per town by admins. This pull request adds a new admin town command toggle for upkeep.
____
#### New Nodes/Commands/ConfigOptions: 

`/ta town <town> toggle upkeep`

____
#### Relevant Towny Issue ticket:

____
- [x] I have tested this pull request for defects on a server. 

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java [hunk 1]
```diff
@@ -1333,6 +1333,11 @@ public void parseAdminTownCommand(String[] split) throws TownyException {
 					town.setHasUnlimitedClaims(choice.orElse(!town.hasUnlimitedClaims()));
 					town.save();
 					TownyMessaging.sendMsg(sender, Translatable.of("msg_town_unlimitedclaims_setting_set_to", town.getName(), town.hasUnlimitedClaims()));
+				} else if (split[2].equalsIgnoreCase("upkeep")) {
+					
+					town.setHasUpkeep(!town.hasUpkeep());
```
Comment (LlmDl) [inline src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java:1338]:
```suggestion
					town.setHasUpkeep(choice.orElse(!town.hasUpkeep()));

```

**Review by LlmDl — DISMISSED** [review]
Will you also be adding a feature for nation upkeep? You included a lang string for them.

**Comment by Gamer153** [comment]
I wanted to add this for nations too, but nations can't override the towny config setting, so I thought it isn't really necessary. Also, I've implemented the change you suggested.

**Comment by LlmDl** [comment]
You'll need to rebase this branch, the en-US has had changes since it was begun.

**Review by Warriorrrr — APPROVED** [review]

**Pull request merged**